<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>195</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>195</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>352</width>
    <height>195</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Not Hesaplayıcı</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>331</width>
      <height>151</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>308</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="not_hesapla_toolButton">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Harf Notunu
Hesapla</string>
        </property>
        <property name="icon">
         <iconset resource="../NotHesaplayici.qrc">
          <normaloff>:/resimler/not.png</normaloff>:/resimler/not.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="shortcut">
         <string>1</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="ortalama_hesapla_toolButton">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ortalama
Hesapla</string>
        </property>
        <property name="icon">
         <iconset resource="../NotHesaplayici.qrc">
          <normaloff>:/resimler/ortalama.png</normaloff>:/resimler/ortalama.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="shortcut">
         <string>2</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>352</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuDosya">
    <property name="title">
     <string>Dosya</string>
    </property>
    <addaction name="actionCikis"/>
   </widget>
   <widget class="QMenu" name="menuYardim">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Yardım</string>
    </property>
    <addaction name="actionLisans"/>
    <addaction name="separator"/>
    <addaction name="actionHakkinda"/>
    <addaction name="actionQtHakkinda"/>
   </widget>
   <addaction name="menuDosya"/>
   <addaction name="menuYardim"/>
  </widget>
  <action name="actionCikis">
   <property name="icon">
    <iconset resource="../NotHesaplayici.qrc">
     <normaloff>:/resimler/cikis.png</normaloff>:/resimler/cikis.png</iconset>
   </property>
   <property name="text">
    <string>Çıkış</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionHakkinda">
   <property name="text">
    <string>Hakkında</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionQtHakkinda">
   <property name="text">
    <string>Qt Hakkında</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionLisans">
   <property name="text">
    <string>Lisans</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../NotHesaplayici.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>hakkindaDialog()</slot>
 </slots>
</ui>
